src/vcbuild/dmd_backend.vcxproj.filters

```diff
@@ -174,15 +174,9 @@
     <ClCompile Include="..\dmd\tk\filespec.c">
       <Filter>dmd\tk</Filter>
     </ClCompile>
-    <ClCompile Include="..\dmd\tk\list.c">
-      <Filter>dmd\tk</Filter>
-    </ClCompile>
     <ClCompile Include="..\dmd\tk\mem.c">
       <Filter>dmd\tk</Filter>
     </ClCompile>
-    <ClCompile Include="..\dmd\tk\vec.c">
-      <Filter>dmd\tk</Filter>
-    </ClCompile>
```

Comment:
It might make sense to move the `vec.h` header file to the backend, s.t. D code and header files are "in sync".